layouts/get-started/get-started.html.html

```diff
@@ -204,74 +204,77 @@ <h2>
+                            Identify patients with or at risk for for substance use disorders 
                         </span>
-                        <p>
-                            <a href="/identify/" class="btn btn-secondary btn-lg">Enter Domain</a> 
-                        </p>
+
+                        <a href="/domains/identify/" class="btn btn-secondary btn-lg enter-domain">Enter Domain</a> 
+
                         <p>Learn how to:</p>
+                        <span class="domain-sub-head"><strong>Screen</strong></span>
                         <p>
-                            <em>Screen</em>- Screen for substance disorders and patients at risk in the dental setting. 
+                            Screen for substance disorders and patients at risk in the dental setting. 
                         </p>
+                        <span class="domain-sub-head"><strong>Use Tools</strong></span>
                         <p>
-                            <em>Use Tools</em>- Demonstrate awareness of different opioid risk assessment and drug monitoring tools that help to identify at-risk patients. 
+                            Demonstrate awareness of different opioid risk assessment and drug monitoring tools that help to identify at-risk patients. 
                         </p>
+                        <span class="domain-sub-head"><strong>Take Charge</strong></span>
                         <p>
-                            <em>Take Charge</em>- Appreciate the importance a dental provider's role in screening for substance disorders and patients at-risk. 
+                            Appreciate the importance a dental provider's role in screening for substance disorders and patients at-risk. 
                         </p>
                     </div>
                     <div class="col-md-4 domains">
                         <div class="domain-logo">
-                            <a href="/refer/" aria-label="Start the Refer domain">{{ partial "icon-feature-refer.html" }}</a> 
+                            <a href="/domains/refer/" aria-label="Start the Refer domain">{{ partial "icon-feature-refer.html" }}</a> 
                         </div>
-                        <h3 class="domain-title">
-                            Refer 
-                        </h3>
+                        <h3 class="domain-title">Refer</h3>
                         <span class="get-started-domain-sub text-center">
-                            Refer patients<br />substance use<br />disorders. 
+                            Refer patients substance use&nbsp;disorders 
                         </span>
-                        <p>
-                            <a href="/refer/" class="btn btn-secondary btn-lg">Enter Domain</a> 
-                        </p>
+
+                        <a href="/domains/refer/" class="btn btn-secondary btn-lg enter-domain">Enter Domain</a> 
+
                         <p>Learn how to:</p>
```

Comment:
Note: I suggest removing these long columns full of text. They are hard to read. Once the user is within the domain, they will get a more detailed understanding.